src/components/UserInfo/UserInfo.tsx

```diff
@@ -15,41 +15,43 @@ export interface Props {
   field: string;
   career: string;
   mbti: string;
+  title?: string;
 }
 
-function DetailLeaderInfo({
+function UserInfo({
   profileImageUrl,
   nickname,
   field,
   career,
   mbti,
+  title,
 }: Props) {
   return (
-    <LeaderInfoContainer>
-      <Typography variant='h5'>리더 정보</Typography>
-      <LeaderInfoWrapper>
+    <UserInfoContainer>
+      {title && <Typography variant='h5'>{title}</Typography>}
+      <UserInfoWrapper>
         {typeof profileImageUrl === 'string' && profileImageUrl ? (
-          <LeaderProfileImageWrapper>
-            <LeaderProfileImage src={profileImageUrl} alt='profile-image' />
-          </LeaderProfileImageWrapper>
+          <UserProfileImageWrapper>
+            <UserProfileImage src={profileImageUrl} alt='profile-image' />
+          </UserProfileImageWrapper>
         ) : (
           <AccountCircleIcon fontSize='large' color='secondary' />
         )}
         <div>
           <Typography variant='h6'>{nickname}</Typography>
-          <LeaderSubInfoWrapper>
+          <UserSubInfoWrapper>
             <Typography variant='subtitle2' color='#6B7280'>
               {field}
             </Typography>
             <Typography variant='subtitle2' color='#6B7280'>
```

Comment:
`font-weight`를 확인해보니 `Typography` 컴포넌트의 `variant`가 `body1`으로 되어야 하는 것 같습니다!
```suggestion
            <Typography variant='body1' color='#6B7280'>
              {field}
            </Typography>
            <Typography variant='body1' color='#6B7280'>
```